templates/reset_password.hbs

```diff
@@ -0,0 +1,11 @@
+<html>
+  <head>
+    <title>Reset Password</title>
+  </head>
+  <body>
+    <div>
+      <p>Here's the token to reset your ScoutTrek password:</p>
```

Comment:
Could we add some additional text saying something like "If you did not make this request please ignore it"?